python/servo/devtools_tests.py

```diff
@@ -572,16 +563,15 @@ def test_source_content_inline_script_with_domparser(self):
     # Test case that uses XMLHttpRequest#responseXML and would actually need the HTML parser
     # (innerHTML has a fast path for values that don’t contain b'&' | b'\0' | b'<' | b'\r')
     def test_source_content_inline_script_with_responsexml(self):
-        self.start_web_server(test_dir=self.get_test_path("sources_content_with_responsexml"))
-        self.run_servoshell()
+        self.run_servoshell(url=f"{self.base_urls[0]}/sources_content_with_responsexml/test.html")
         expected_content = open(self.get_test_path("sources_content_with_responsexml/test.html")).read()
-        self.assert_source_content(Source("inlineScript", f"{self.base_urls[0]}/test.html"), expected_content)
+        self.assert_source_content(Source("inlineScript", f"{self.base_urls[0]}/sources_content_with_responsexml/test.html"), expected_content)
 
     def test_source_breakable_lines_and_positions(self):
-        self.start_web_server(test_dir=self.get_test_path("sources_breakable_lines_and_positions"))
-        self.run_servoshell()
+        # self.start_web_server(test_dir=self.get_test_path("sources_breakable_lines_and_positions"))
```

Comment:
I think you have a stray commented line here?

Reply:
yep, removed in 57fc7145af8ea9c6894eea2c783bbdb96d4512bb